### PR TITLE
ISPN-10987 Xsite simple tutorial

### PR DESCRIPTION
**Pull request by karesti**
https://issues.jboss.org/browse/ISPN-10987

### CODE REVIEW

**Comment by galderz** [comment]
Looking forward to trying it out when the docs are provided.

**Comment by karesti** [comment]
@oraNod the tutorial is ready

File: xsite-replication/xsiteCache.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsiteCache" statistics="true">
+      <backups>
+        <backup site="NYC" strategy="SYNC" failure-policy="WARN" timeout="12003">
+          <state-transfer chunk-size="098" timeout="7654" max-retries="321" wait-time="0101"/>
```
Comment (pruivo) [inline xsite-replication/xsiteCache.xml]: this can be removed as well... too many random values xD

Comment (karesti) [reply xsite-replication/xsiteCache.xml]: the state transfert stuff ?

File: xsite-replication/xsiteCache.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<infinispan>
+  <cache-container>
+    <replicated-cache name="xsiteCache" statistics="true">
+      <backups>
+        <backup site="NYC" strategy="SYNC" failure-policy="WARN" timeout="12003">
```
[the remark this answers is not in the document]
Comment (karesti) [reply xsite-replication/xsiteCache.xml]: ok

File: xsite-replication/xsiteCache.xml (hunk 1 above)
Comment (karesti) [reply xsite-replication/xsiteCache.xml]: you mean the backup property or the state transfert line

Comment (pruivo) [reply xsite-replication/xsiteCache.xml]: the <state-transfer ..>

Comment (karesti) [reply xsite-replication/xsiteCache.xml]: ok

**Comment by karesti** [comment]
@pruivo done

**Comment by pruivo** [comment]
integrated! thanks @karesti !

**Pull request merged**